query/src/sessions/query_ctx.rs

```diff
@@ -424,6 +430,30 @@ impl QueryContext {
         })?;
         Ok(FunctionContext { tz })
     }
+
+    pub fn block_on_meta<F>(&self, future: F) -> Result<F::Output>
+    where
```

Comment:
With this function, `show function ` can work well in Mac Osx.


I do not know whether this code is right or not.  cc @drmingdrmer  @dantengsky